pyproject.toml

```diff
@@ -27,6 +27,7 @@ tiktoken = ">=0.7.0"
 importlib-metadata = ">=6.8.0"
 tokenizers = "*"
 click = "*"
+rich = ">=13.0.0"
```

Comment:
please add this as optional, see uvicorn example below

Reply:
this should be in `[tool.poetry.extras]`

Reply:
> this should be in `[tool.poetry.extras]`

Fixed in 2da6e31ad.